apps/minifront/src/components/swap/swap-form/simulate-swap-result/traces/index.tsx

```diff
@@ -29,17 +29,14 @@ export const Traces = ({
 
       <div className='mt-4 flex overflow-auto [scrollbar-width:thin]'>
         <div className='mx-2 w-min grow'>
-          <div className='-mx-2 -mb-8 flex justify-between'>
-            <div className='flex flex-col items-start gap-2'>
-              <ValueViewComponent view={input} size='sm' />
-              <ArrowDown size={17} className='relative z-10' />
-            </div>
-            <div className='flex flex-col items-end gap-2'>
-              <ValueViewComponent view={output} size='sm' />
-              <ArrowUp size={17} className='relative z-10' />
+          <div className='relative flex items-center justify-between'>
+            <ValueViewComponent view={input} size='sm' />
+            <div className='absolute left-1/2 -translate-x-1/2 flex items-center'>
+              <ArrowRight size={20} strokeWidth={2.5} className='text-white' />
```

Comment:
thoughts about replacing the up and down arrows with a single right arrow? the up and down arrows had alignment issues as the routes expanded

-------------

_before_

<img width="738" alt="Screenshot 2024-11-21 at 7 49 07 PM" src="https://github.com/user-attachments/assets/c045ff7b-3ec4-4528-8dcf-64df3765e9b9">

-------------

_after_

<img width="739" alt="Screenshot 2024-11-21 at 7 49 33 PM" src="https://github.com/user-attachments/assets/6c6ccced-a133-48d4-ab9a-6895114a9bb1">